drivers/unix/net_socket_posix.cpp

```diff
@@ -739,15 +739,15 @@ bool NetSocketPosix::is_open() const {
 	return _sock != SOCK_EMPTY;
 }
 
-int NetSocketPosix::get_available_bytes() const {
+uint32_t NetSocketPosix::get_available_bytes() const {
 	ERR_FAIL_COND_V(!is_open(), -1);
```

Comment:
Still returning -1 here

Reply:
Good catch.
StreamPeerMbedTLS::get_available_bytes was already returning 0 for error.
To me it makes sense to return 0 in that case + log because it means there is no bytes available.

Other solution could be to return Error and have the number of bytes returned as out parameter. Since this changes the API its worth doing right now.

What do you think ?

Reply:
> Since this changes the API its worth doing right now.

That would be a much larger API breakage and not comparable with changing the return type (and won't work with GDScript)